js/whitebit.js

```diff
@@ -1162,14 +1177,29 @@ module.exports = class whitebit extends Exchange {
             throw new ExchangeError (this.id + ' ' + code.toString () + ' endpoint not found');
         }
         if (response !== undefined) {
-            const status = this.safeValue (response, 'status');
-            if ((status !== undefined && status !== '200')) {
+            // For cases where we have a meaningful status
+            // {"response":null,"status":422,"errors":{"orderId":["Finished order id 435453454535 not found on your account"]},"notification":null,"warning":"Finished order id 435453454535 not found on your account","_token":null}
+            const status = this.safeInteger (response, 'status');
+            // For these cases where we have a generic code variable error key
+            // {"code":0,"message":"Validation failed","errors":{"amount":["Amount must be greater than 0"]}}
+            const code = this.safeInteger (response, 'code');
+            const hasErrorStatus = status !== undefined && status !== '200';
+            if (hasErrorStatus || code !== undefined) {
```

Comment:
If the status is present I'm using it to identify the exception since it's unique per exception. Otherwise, I'm using the error description because the "code" represents an error category (validation failed for instance) that might be common to several different exceptions.

@kroitor Please let me know if there is a better way to handle this